hadoop-ozone/common/src/main/bin/ozone

```diff
@@ -55,6 +55,7 @@ function hadoop_usage
   hadoop_add_subcommand "version" client "print the version"
   hadoop_add_subcommand "dtutil" client "operations related to delegation tokens"
   hadoop_add_subcommand "upgrade" client "HDFS to Ozone in-place upgrade tool"
+  hadoop_add_subcommand "omha" client "OM HA tool"
```

Comment:
NIT: Have we named this command `omha` because we have plans in future to also add similar command for SCM?
If not, how about naming the command as `haadmin` , `omadmin`, `admin`.
Not a deal breaker but was just thinking if we can keep it similar to hdfs.

Reply:
Changed the command to "ozone admin om". 
SCM HA is also in the pipeline sometime in the future. 
As Marton also suggested, we can have all admin commands under this. This will include SCMCli also.